README.md

```diff
@@ -194,12 +194,23 @@ value` mapping e.g.
 ```toml
 avoid-breaking-exported-api = false
 disallowed-names = ["toto", "tata", "titi"]
-cognitive-complexity-threshold = 30
 ```
 
 See the [list of configurable lints](https://rust-lang.github.io/rust-clippy/master/index.html#Configuration),
 the lint descriptions contain the names and meanings of these configuration variables.
 
+See [table of lint configurations](https://doc.rust-lang.org/nightly/clippy/lint_configuration.html)
+to see what configuration options you can set and the lints they configure.
```

Comment:
I remember the first reference to Clippy's lint list was mainly added to make people more aware. The documentation added by this PR is better for that. Maybe we can join these two together, saying something like this:

The [table of configurations](https://doc.rust-lang.org/nightly/clippy/lint_configuration.html) contains all config values, their default, and a list of lints they affect. Each [configurable lint](https://rust-lang.github.io/rust-clippy/master/index.html#Configuration), also contains information about these values.

```
The [table of configurations](https://doc.rust-lang.org/nightly/clippy/lint_configuration.html)
contains all config values, their default, and a list of lints they affect.
Each [configurable lint](https://rust-lang.github.io/rust-clippy/master/index.html#Configuration)
, also contains information about these values.
```

Reply:
I made this change.